lib/testing/gpuSimulateN2k.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef GPU_SIMULATE_N2K_HPP
+#define GPU_SIMULATE_N2K_HPP
+
+#include "Config.hpp"          // for Config
+#include "Stage.hpp"           // for Stage
+#include "bufferContainer.hpp" // for bufferContainer
+
+#include <stdint.h> // for int32_t
+#include <string>   // for string
+
+class gpuSimulateN2k : public kotekan::Stage {
```

Comment:
This needs documentation with a description, and list of input/output buffers, and config options.   The header descriptions are very useful for constructing the config files.